install/gcp/installer/configs/config.py

```diff
@@ -34,6 +34,8 @@ def __init__(self, **kwargs):
                               datetime.datetime.now().strftime('%Y%m%d%H%M%S'))
         self.identifier = None
         self.force_no_cloudshell = bool(kwargs.get('no_cloudshell'))
+        self.service_account_key_path = kwargs.get(
+            'service_account_key_path') or None
```

Comment:
Don't need the `or None` here as the flag will already default to None.  Also, `some_dict.get('foo')` will also default to `None`.

Reply:
Done